griptape/memory/structure/conversation_memory.py

```diff
@@ -17,6 +17,7 @@ class ConversationMemory:
     runs: list[Run] = field(factory=list, kw_only=True)
     structure: Structure = field(init=False)
     autoload: bool = field(default=True, kw_only=True)
+    buffer_size: Optional[int] = field(default=None, kw_only=True)
```

Comment:
Should be also added to `ConversationMemorySchema`.

Reply:
You can also remove it from:
1. `BufferConversationMemory`.
2. `BufferConversationMemorySchema`.

Reply:
And, finally, `SummaryConversationMemory.offset` can be swapped out as well.

Reply:
3. Makes sense. Let's rename `buffer_size` to `max_memory_runs` and leave `offset`. Unless we want to rename `offset` into something more meaningful.